docusaurus/docs/extensions/publishing.md

```diff
@@ -42,6 +42,113 @@ Running any of the commands above will only build your images - in order to publ
 yarn publish-pkgs -r <REGISTRY> -o <ORGANIZATION> -p -c
 ```
 
-## Additional Information
+## Additional Release Configuration
```

Comment:
My opinion here is that we are missing what the chapter `Running the Publish Script` brought in a way: some clear cut examples on how to run the publish pkg script for both packages and containers. What do you think @jordojordo ?

Reply:
You're right, there was not any clear examples on running the publish script for the package chart builds. I have added those steps, let me know what you think.